autotyping/autotyping.py

```diff
@@ -513,35 +554,49 @@ def type_of_expression(expr: libcst.BaseExpression) -> Optional[Type[object]]:
     """
     if isinstance(expr, libcst.Float):
         return float
-    elif isinstance(expr, libcst.Integer):
+    if isinstance(expr, libcst.Integer):
```

Comment:
These changes are unnecessary, please leave the existing style

Reply:
I've added `method = ...` before one of the `elif` parts, which forced me to convert that part into regular `if`. I can leave the rest intact but then it will be inconsistent. Also, I wanted to highlight that each of the branches has a guaranteed early return, and so the `method = ...` part won't be executed for any of them. Now, with this in mind, which version would you prefer? A mix of `if` and `elif` or the new `if` only version?

Reply:
Also, the following addition uses double `if`:

```python
    if isinstance(expr, libcst.Comparison) and len(expr.comparisons) == 1:
        operator = expr.comparisons[0].operator
        if isinstance(operator, libcst.Is):
            return bool
```

That means, if the later conditions use `elif`, then failing the second check there will lead to the rest of the checks being skipped. It might lead to unintended behavior when one of the later branches was supposed to match but was unintentionally skipped.

Reply:
OK, we can leave this as is.